src/grpc/stream_infer_handler.cc

```diff
@@ -189,7 +189,7 @@ ModelStreamInferHandler::Process(InferHandler::State* state, bool rpc_ok)
       state->context_->responder_->Finish(status, state);
       return !finished;
     }
-
+    state->context_->ExtractStateFromHeaders(state);
```

Comment:
Can this be treated as a part of `ExecutePrecondition` logic, or we would like to keep it separate?

Reply:
Makes sense to include as part of `ExecutePrecondition`